Formula/d/dug.rb

```diff
@@ -0,0 +1,22 @@
+class Dug < Formula
+  desc "Global DNS propagation checker that gives pretty output"
+  homepage "https://dug.unfrl.com"
+  url "https://github.com/unfrl/dug/archive/refs/tags/0.0.93.tar.gz"
+  sha256 "a375034bcad3193d47a485b3dd33c7633f7890e5f79b6a9b6b3b3bc198fb2ec3"
+  license "MIT"
+
+  depends_on "dotnet"
+  uses_from_macos "zlib"
+
+  def install
+    ENV["DOTNET_SYSTEM_GLOBALIZATION_INVARIANT"] = "1"
+    system "dotnet", "publish", "-c", "Release", "-p:PublishSingleFile=true",
+            "-p:IncludeNativeLibrariesForSelfExtract=true", "--self-contained",
+            "true", "-o", "publish", "./cli"
+    bin.install "./publish/dug"
```

Comment:
might be better to build like other dotnet formulae

```
    dotnet = Formula["dotnet"]
    os = OS.mac? ? "osx" : OS.kernel_name.downcase
    arch = Hardware::CPU.intel? ? "x64" : Hardware::CPU.arch.to_s

    args = %W[
      --configuration Release
      --framework net#{dotnet.version.major_minor}
      --output #{libexec}
      --runtime #{os}-#{arch}
      --no-self-contained
      -p:PublishSingleFile=true
      -p:IncludeNativeLibrariesForSelfExtract=true
    ]

    system "dotnet", "publish", "cli/dug.csproj", *args
    env = { DOTNET_ROOT: "${DOTNET_ROOT:-#{dotnet.opt_libexec}}" }
    (bin/"dug").write_env_script libexec/"dug", env
```

Reply:
Nice.